doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -1068,6 +1068,14 @@ isvoice <nickname> [channel]
 
   Module: irc
 
+^^^^^^^^^^^^^^^^^
+isaway <nickname>
```

Comment:
this should be `isaway <nickname> [channel]` like all the other commands

Reply:
isaway is a global state, not on a per-channel basis. I can't be away on one channel, but not another. It is server-level.

Reply:
we do not track users server-wide. that data structure does not exist in eggdrop. we track channelusers.